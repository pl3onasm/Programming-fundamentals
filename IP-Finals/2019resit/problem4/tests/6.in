20 -2*x^10-5*x^0+1*x^10+3*x^3-2*x^4-4*x^2-4*x^7-5*x^2-1*x^1-1*x^4-1*x^1+1*x^6+4*x^7-3*x^5+2*x^6+2*x^3+3*x^1-1*x^0-2*x^6+1*x^2
