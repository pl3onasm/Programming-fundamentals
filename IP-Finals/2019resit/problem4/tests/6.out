(((((((((-1*x+0)*x+0)*x+0)*x+1)*x-3)*x-3)*x+5)*x-8)*x+1)*x-6
